### ISSUE TITLE
Need documentation on exporting data via TopoJSON

### ISSUE THREAD

**Issue by alukach** [cugos/dropchop#241]
We need documentation in README.md for how a user can export data as TopoJSON from Dropchop.

**Comment by alukach** [comment]
@jczaplew Interested in adding this?

**Comment by jczaplew** [comment]
Yep! I can take care of it, along with #240 which is very similar

**Comment by mapsam** [comment]
@jczaplew you should link to your blog post as well, it was a fascinating deep dive.
